Update Documentation (#7)
ministryofjustice/pttp-infrastructure: documentation/deployment.md

# Deployment

## Environments

There are currently 3 AWS environments that run this service:
1. Development
2. Pre-production
3. Production

> These are deployed from an additional environment known as Shared Services.

Reviewer: ```suggestion
The environments are deployed using an additional environment known as [Shared Services](https://github.com/ministryofjustice/pttp-shared-services-infrastructure).
```
